README.md

```diff
@@ -659,14 +659,15 @@ end
 ## Testing with CarrierWave
 
 It's a good idea to test your uploaders in isolation. In order to speed up your
-tests, it's recommended to switch off processing in your tests, and to use the
+tests, it's recommended to switch off processing in your tests, disable SSRF protection, and to use the
```

Comment:
Actually I don't recommend to do this. The users shouldn't do this without knowing what they're doing, unless they will be at risk.
Could you rephrase this part?